app/src/full/java/io/homeassistant/companion/android/notifications/MessagingService.kt

```diff
@@ -423,7 +423,9 @@ class MessagingService : FirebaseMessagingService() {
             builder.setContentTitle(getSpannedTextFromHtml(it))
         }
         data[MESSAGE]?.let {
-            builder.setStyle(NotificationCompat.BigTextStyle().bigText(getSpannedTextFromHtml(it)))
+            val text = getSpannedTextFromHtml(it)
+            builder.setContentText(text)
```

Comment:
Won't this override what is set in `handleSubject` above?

Reply:
handleSubject is called after handleText. So the subject should overwrite the content text, if defined